src/random_test.cpp

```diff
@@ -1215,7 +1216,7 @@ bool execute_sql(std::string sql, Thd1 *thd) {
     if (log_all || log_success) {
       thd->thread_log << " S " << sql;
       if (result == NULL) {
-        thd->thread_log << std::endl;
+        thd->thread_log << " affected_rows:" << affected_rows_count << std::endl;
```

Comment:
use abbreviations instead of long string affected_rows.
else It would increase the size of the log file.

Reply:
check if you can directly call the method.
thd->thread_log <<  mysql_affected_rows(thd->conn) << std::endl;

Reply:
move that also inside the if block.